src/views/catalog/customize/tests/data/projects.json

```diff
@@ -0,0 +1,206 @@
+          "include.release.openshift.io/single-node-developer": "true",
+          "openshift.io/node-selector": "",
+          "openshift.io/sa.scc.mcs": "s0:c15,c5",
+          "openshift.io/sa.scc.supplemental-groups": "1000220000/10000",
+          "openshift.io/sa.scc.uid-range": "1000220000/10000",
+          "workload.openshift.io/allowed": "management"
+        },
+        "ownerReferences": [
+          {
+            "apiVersion": "config.openshift.io/v1",
+            "kind": "ClusterVersion",
+            "name": "version",
+            "uid": "5ae272e8-6b51-4a4d-a268-229e3b8bf506"
+          }
+        ],
+        "managedFields": [
+          {
+            "manager": "cluster-policy-controller",
+            "operation": "Update",
+            "apiVersion": "v1",
+            "time": "2022-01-27T00:58:22Z",
+            "fieldsType": "FieldsV1",
+            "fieldsV1": {
+              "f:metadata": {
+                "f:annotations": {
+                  "f:openshift.io/sa.scc.mcs": {},
+                  "f:openshift.io/sa.scc.supplemental-groups": {},
+                  "f:openshift.io/sa.scc.uid-range": {}
+                }
+              }
+            }
+          },
+          {
+            "manager": "cluster-version-operator",
+            "operation": "Update",
+            "apiVersion": "v1",
+            "time": "2022-01-27T00:58:22Z",
+            "fieldsType": "FieldsV1",
+            "fieldsV1": {
+              "f:metadata": {
+                "f:annotations": {
+                  ".": {},
+                  "f:include.release.openshift.io/ibm-cloud-managed": {},
+                  "f:include.release.openshift.io/self-managed-high-availability": {},
+                  "f:include.release.openshift.io/single-node-developer": {},
+                  "f:openshift.io/node-selector": {},
+                  "f:workload.openshift.io/allowed": {}
+                },
+                "f:labels": {
+                  ".": {},
+                  "f:kubernetes.io/metadata.name": {},
+                  "f:openshift.io/cluster-monitoring": {},
+                  "f:openshift.io/run-level": {}
+                },
+                "f:ownerReferences": {
+                  ".": {},
+                  "k:{\"uid\":\"5ae272e8-6b51-4a4d-a268-229e3b8bf506\"}": {}
+                }
+              }
+            }
+          },
+          {
+            "manager": "olm",
+            "operation": "Update",
+            "apiVersion": "v1",
+            "time": "2022-01-27T01:06:11Z",
+            "fieldsType": "FieldsV1",
+            "fieldsV1": {
+              "f:metadata": {
+                "f:labels": {
+                  "f:olm.operatorgroup.uid/be5e23b9-c849-45c4-9eca-6404b4a5bea7": {}
+                }
+              }
+            }
+          }
+        ]
+      },
+      "spec": {
+        "finalizers": [
+          "kubernetes"
+        ]
+      },
+      "status": {
+        "phase": "Active"
+      }
+    }
+]
```

Comment:
missing line ending for all jsons

Reply:
should we add a new rule in eslint? https://eslint.org/docs/rules/eol-last